lib/index.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const autobind = require('auto-bind');
+const path = require('path');
+
+const defaultOptions = { ttl: 0 };
+
+class CacheManager {
+  constructor(options = defaultOptions) {
```

Comment:
You can maybe use `Object.assign` or `_.defaults` to assign default options so that consumers can pass other options and still leverage default `ttl`.

Reply:
Yeah, I agree. I fixed it here: https://github.com/ExtensionEngine/tapster/pull/2/files#diff-92bbac9a308cd5fcf9db165841f2d90ce981baddcb2b1e26cfff170929af3bd1R10 but forgot to update it in this PR. 
Just one note, I used spread operator for this purpose because `Object.assign` is not fitting this case well. 
For example, if I have this case:
```js
const options = { ttl: 10 };
Object.assign(options, defaultOptions); 
```
The result would be that the `options` object is equal to: `{ ttl: 0 }`.
Also, if I change the order of arguments such as:
```js
const options = { ttl: 10 };
Object.assign(defaultOptions, options); 
```
it would alter the `defaulOptions` object, which would result in future bugs.

Reply:
I can make the `defaultOptions` object to be locally defined inside the constructor. By doing it this way, it will also work with Object.assign 🤦 .
Is there any company standard for this case or I can use both spread operator and `Object.assign`?

Reply:
Defaulting via `Object.assign` is done by using empty object as first argument i.e. `Object.assign({}, defaults, options)` but spread also works. Whatever you prefer is fine.

Reply:
Yeah, you are right! Thanks <3